src/lib/server/region.ts

```diff
@@ -78,3 +78,8 @@ export const getProjectFromUserState = (userState: UserState, projectName?: stri
 	if (!projectData) error(404, `Project "${projectName}" not found`);
 	return projectData;
 };
+
+export const invalidateStrategyForProject = async (userState: UserState, projectName: string) => {
```

Comment:
whenever we make changes to region by running model we need to invalidate strategize data. this is because the cost ranges + cases data is no longer valid